### PR TITLE
Plugin implementation

### PR DESCRIPTION
**Pull request by avkos**

### CODE REVIEW

File: package.json
Comment (jdevcs) [inline package.json]: this will need rename to `web3-plugin-blob-tx` accordign to new repo

File: src/index.ts
Comment (jdevcs) [inline src/index.ts]: rename here as well

Comment (jdevcs) [reply src/index.ts]: and other places in other files

File: package.json
Comment (jdevcs) [inline package.json:23]: update required for : https://github.com/web3/web3.js-plugin-blob-tx

File: src/index.ts
Comment (jdevcs) [inline src/index.ts]:
```suggestion
export class Web3BlobTxPlugin extends Web3PluginBase {
```

File: test/index.test.ts
Comment (jdevcs) [inline test/index.test.ts:4]: could you also add local E2E test for sepolia NW for sending blob tx test with PK in env var

**Comment by avkos** [comment]
Build will not work until we not merge and make new realse with [these changes](https://github.com/web3/web3.js/pull/7000)

After this I need to change version of web3

File: src/plugin.ts
Comment (jdevcs) [inline src/plugin.ts:290]: support of `getTransaction` and `getTransactionFromBlock` for blob tx is also required

**Review by jdevcs — COMMENTED** [review]
lgtm, except after merging PR in main web3.js tests should be passing for merging this PR. Thanks @avkos

File: src/plugin.ts
Comment (jdevcs) [inline src/plugin.ts]: this should be 0x3 https://eips.ethereum.org/EIPS/eip-4844

**Review by jdevcs — APPROVED** [review]

**Pull request merged**
